content/05kapitel.tex

```diff
@@ -320,6 +320,108 @@ \subsection{Figma Designs}
+    \label{fig:umlActivityOverview.png}
+\end{figure}
+
+In Abbildung~\ref{fig:umlActivityFeed.png} werden die möglichen Aktionen auf der Feedseite dargestellt.
+Der aufgerufene Feed kann aus den Abonnement ausgetragen werden. Im Feed kann gefiltert und/ oder gesucht werden.
+Einzelne Artikel können als Lesezeichen gespeichert oder geliked werden. Per Klick auf den Artikel kann dieser aufgerufen werden.
+\begin{figure}
+    \includegraphics[width=\linewidth]{umlActivityFeed.png}
+    \caption{Aktivitätsdiagramm der Feedseite}
+    \label{fig:umlActivityFeed.png}
+\end{figure}
+
+Auf der Artikelseite kann wie in Abbildung~\ref{fig:umlActivityView.png} dargestellt der Artikel gelesen werden.
+Darüber hinaus kann der Artikel als Lesezeichen gespeichert, geliked und/oder geteilt werden.
+Auch die Feedseite des Artikels kann aufgerufen werden. Die Ursprungswebseite des Artikels kann auch geöffnet werden.
+Unter den ähnlichen Artikeln kann zu diesem per Klick gewechselt werden.
+\begin{figure}
+    \includegraphics[width=\linewidth]{umlActivityView.png}
+    \caption{Aktivitätsdiagramm der Artikelseite}
+    \label{fig:umlActivityView.png}
+\end{figure}
+
+
+In Abbildung~\ref{fig:umlActivityBookmarks.png} ist das Aktivitätsdiagramm der Lesezeichenseite dargestellt.
+Alle vom Nutzer gespeicherten Lesezeichen werden hier aufgelistet und es kann gefiltert und/ oder darin gesucht werden.
+Einzelne Artikel können aus den Lesezeichen gelöscht werden. Auch können einzelne Artikel geliked oder entliked werden.
+\begin{figure}
+    \includegraphics[width=\linewidth]{umlActivityBookmarks.png}
+    \caption{Aktivitätsdiagramm der gespeicherten Lesezeichenseite}
+    \label{fig:umlActivityBookmarks.png}
+\end{figure}
+
+Die Aktivitäten der Abonnementsseite werden in Abbildung~\ref{fig:umlActivitySubscription.png} abgebildet.
+Die Abonnements können gefiltert und/oder darin gesucht werden. Jeder abonnierte Feed kann abgewählt werden.
+Ein Feed kann ausgeklappt werden und einzelne Artikel geliked oder als Lesezeichen gespeichert werden.
+Die Artikel können mit einem Klick aufgerufen werden.
+\begin{figure}
+    \includegraphics[width=\linewidth]{umlActivitySubscription.png}
+    \caption{Aktivitätsdiagramm der Abonnements}
+    \label{fig:umlActivitySubscription.png}
+\end{figure}
+
+In der Abbildung~\ref{fig:umlActivityAddFeed.png} wird der Ablauf zum Hinzufügen eines neuen Feeds dargestellt.
+Ein RSS-Link wird eingetragen und mit klick auf "load" wird der Link geprüft. Ist der Link fehlerbehaftet muss er korrigiert werden.
+Funktioniert der Link werden Name und Beschreibung vorausgefüllt.
+Der Name und die Beschreibung kann vor dem finalen hinzufügen noch angepasst werden.
+Das Hinzufügen eines Feeds kann jederzeit über die Schaltfläche verlassen werden.
+\begin{figure}
+    \includegraphics[width=\linewidth]{umlActivityAddFeed.png}
+    \caption{Aktivitätsdiagramm zum Hinzufügen eines Feeds}
+    \label{fig:umlActivityAddFeed.png}
```

Comment:
Rss feed invalid (bspw. bei falscher formatierung) ist auch möglich, bitte in diragramm  adden

Reply:
3 option weil rss funktioniert aber nicht mit unsrem schema

Reply:
[x]

Reply:
Es gibt keinen eg um von enter rss link zu exit pop up zu gehen, müsste evtl. noch geadded werden

Reply:
[x]